domain-get-subdomains/index.ts

```diff
@@ -1,11 +1,43 @@
 import { AzureFunction, Context, HttpRequest } from "@azure/functions";
-import { doServiceOperation, getSubdomains } from "../src/services/domainService";
+import { DomainId } from "@zero-tech/data-store-core";
+import { getMongoDomainService } from "../src/helpers";
+import { getDomainFindOptionsFromQuery } from "../src/helpers/domainFindOptionsHelper";
+import { validateDomainId } from "../src/schemas";
 
 const httpTrigger: AzureFunction = async function (
   context: Context,
   req: HttpRequest
 ): Promise<void> {
-  await doServiceOperation(getSubdomains, context, req);
+  try {
+    const domainId: DomainId = req.params.id;
+    if (!validateDomainId(domainId)) {
+      context.res = {
+        status: 400,
+        body: validateDomainId.errors,
+      };
+      return;
+    }
+
+    let findOptions = getDomainFindOptionsFromQuery(req);
```

Comment:
Please add a linter to this project and run it with auto fixing enabled. It will convert this to a `const` (or warn you that it can be a `const` instead) You can check out the zdao sdk for an instance of well configured eslint rules for typescript: https://github.com/zer0-os/zdao-sdk

Reply:
Eslint exists on the project already, so ill run it